chat/config-overrides.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function override(config, env) {
+  // Define our own filename
+  config.output.filename = "vscode.js";
```

Comment:
why this is the name?

Reply:
this is just for dev purposes, we can change it